lib/ansible/modules/system/seboolean.py

```diff
@@ -49,7 +49,7 @@
     choices: [ 'yes', 'no' ]
 notes:
    - Not tested on any debian based system
-requirements: [ ]
+requirements: [ libsemanage-python ]
```

Comment:
> IMO this needs to become:

Yeah, you're right.  I hadn't really looked through the code for the module but it will bail out [if either of those isn't available](https://github.com/ansible/ansible/blob/devel/lib/ansible/modules/system/seboolean.py#L178-L182) (L178-L182) so I agree.